src/server.rs

```diff
@@ -81,6 +81,16 @@ pub fn create_server(conf: ConfigOpts, wallet: Wallet<AnyBlockchain, Tree>) -> S
                     Some(address) => address,
                     None => return Ok(response.body(not_found().as_bytes().to_vec())?),
                 };
+                match is_mine_address(&*wallet, address) {
```

Comment:
small English language nit.. I think this and below should be "is_address_mine" or "is_my_address"

Reply:
`is_my_address()` sounds great, thanks